Resources/doc/reference/installation.rst

```diff
@@ -55,10 +56,15 @@ You will need to configure SonataArticleBundle's entities, if you plan to use th
                 article:  Application\Sonata\ArticleBundle\Entity\Article
                 fragment: Application\Sonata\ArticleBundle\Entity\Fragment
 
-            fragments: # this is currently in development, hope we'll get this soon :)
+            fragments:
                 sonata.article.fragment.title:
                 sonata.article.fragment.text:
 
+.. note::
+
+    We plan to improve the fragments available to allow a configuration for each article type.
+    For example, you will want the fragment 'Comments' only on articles of type 'Blog'.
+
```

Comment:
I'm asking b/c I feel that you could have something simpler for this PR like

```
fragments_whitelist:
    - sonata.article.fragment.title
    - sonata.article.fragment.whatever
```

Not thinking about it upfront will create a BC-break, so it would be best to design the configuration right from the start.

Maybe the best design would be to have an interface `FragmentsProvider` with a `getFragmentServicesWhitelist(Article $article)` method ? Then, in the configuration, you can enable whatever implementation you want, including an implementation that does an intersection of what other implementations return : 

```yaml
fragment_whitelist_provider:
    chain_provider:
        simple_array:
            - sonata.article.fragment.comment
            - sonata.article.fragment.whatever
        article_to_fragment:
            sonata.article.type.blog:
                 - comment
        fragment_to_article:
            reviews:
                 - blog
                 - other_article_type
            
```